src/DayDifference.php

```diff
@@ -84,20 +102,26 @@ private function findDifference(int $startStamp, int $endStamp, array $allowedDa
              *
              * @var string $dateFormats
              */
-            $dateFormats = date('w,Y-m-d,*-m-d', $currentStamp);
-
-            /**
-             * @var string $weekDay the number (in string form) representing the current day
-             * @var string $specificDate Y-m-d format of the current day
-             * @var string $wildcardDate *-m-d format of the current day, to allow for exclusions that repeat every year
-             */
-            [$weekDay, $specificDate, $wildcardDate] = explode(',', $dateFormats);
+            if (!isset(self::$cache[$currentStamp])) {
+                $dateFormats = date('w,Y-m-d,*-m-d', $currentStamp);
```

Comment:
You need the `/** @var **/` comment above to be inside the `if` and above line 106 to keep static analysis happy.

Reply:
For some reason, static analysis was happy 🤷.  But I've updated it regardless 👍